plugins/occt/module/vtkF3DOCCTReader.cxx

```diff
@@ -273,7 +303,7 @@ class vtkF3DOCCTReader::vtkInternals
     polydata->SetLines(linesCells);
 
 #if F3D_PLUGIN_OCCT_XCAF
-    /* colors may be left empty if this->ColorTool has not been initialized */
+    /* colors may be left empty ? */
```

Comment:
do you mean that you do not understand why this code is there @snoyer ?

Reply:
there was something about `this->ColorTool` not being initialized (which I wasn't super clear about in the first place) but now I removed it entirely. Would need to investigate what situation this can be reached from...

Reply:
improving the comment in that sense would be nice